CHANGES.md

```diff
@@ -1,3 +1,49 @@
+## edge-19.3.3
+
+**Significant Update** This edge release introduces a new TLS Identity system
+into the default Linkerd installation, replacing `tls=optional` and the
+`linkerd-ca` controller. Now, proxies generate ephemeral private keys into a
+tmpfs directory and dynamically refresh certificates, authenticated by
+Kubernetes ServiceAccount tokens, via the newly-introduced Identity controller.
+
+Now, all meshed HTTP communication is private and authenticated by default.
+
+* CLI
+  * Changed `install` to accept or generate an issuer Secrets for the Identity
```

Comment:
Feels like something is off with the plurals here, "generate an issuer Secrets" - "generates an issuer Secret"?

Reply:
great catch!